src/components/IFrame.tsx

```diff
@@ -86,35 +93,27 @@ function getOldDotURL(url) {
     return pathname;
 }
 
-const propTypes = {
-    // The session of the logged in person
-    session: PropTypes.shape({
-        // The email of the logged in person
-        email: PropTypes.string,
-
-        // The authToken of the logged in person
-        authToken: PropTypes.string,
-    }).isRequired,
-};
-
-function OldDotIFrame({session}) {
+function OldDotIFrame({session}: OldDotIFrameProps) {
     const [oldDotURL, setOldDotURL] = useState('https://staging.expensify.com');
 
     useEffect(() => {
         setOldDotURL(`https://expensify.com.dev/${getOldDotURL(window.location.href)}`);
 
-        window.addEventListener('message', (event) => {
+        window.addEventListener('message', (event: MessageEvent<string>) => {
```

Comment:
Should it be `MessageEvent<string>` or `MessageEvent<Url>`?

Reply:
@VickyStash I guess we can leave it as `string`, since this has to be passed to `getNewDotURL` which receives a `string`